lib/util/DocumentExtractor.js

```diff
@@ -43,8 +43,18 @@ const extractors = ([
       extractFromRequest: request => {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this function to always return the same type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3800)